requirements.txt

```diff
@@ -1,14 +1,14 @@
 # Usage: pip install -r requirements.txt
-Django ~= 3.2.13
+Django ~= 4.2.1
 Jinja2 ~= 3.1.1
 django-jinja ~= 2.10.0
 django-bootstrap-form ~= 3.4
 django-jinja-bootstrap-form ~= 4.3.4
-git+https://github.com/apluslms/django-essentials.git@1.5.0#egg=django-essentials~=1.5.0
+git+https://github.com/apluslms/django-essentials.git@1.6.0#egg=django-essentials~=1.6.0
 git+https://github.com/apluslms/js-jquery-toggle.git@1.1.2#egg=js-jquery-toggle-django~=1.1.2&subdirectory=django
 git+https://github.com/apluslms/django-dictiterators.git@1.1.1#egg=django-dictiterators~=1.1.1
-git+https://github.com/apluslms/django-colortag.git@2.3.1#egg=django-colortag~=2.3.1
-git+https://github.com/apluslms/django-lti-login.git@3.2.0#egg=django-lti-login~=3.2.0
+git+https://github.com/apluslms/django-colortag.git@2.4.0#egg=django-colortag~=2.4.0
+git+https://github.com/apluslms/django-lti-login.git@3.3.0#egg=django-lti-login~=3.3.0
 django-filter ~= 22.1
 django-settingsdict ~= 1.1.1
 celery ~= 5.2.6
```

Comment:
Django 4.2 supports psycopg 3 and it is recommended to upgrade from psycopg2. It doesn't look like there are any compatibility issues in our case so it is easy to install the newer package. Note that psycopg2 and psycopg are different package names in PyPi.

https://docs.djangoproject.com/en/4.2/releases/4.2/#psycopg-3-support
https://pypi.org/project/psycopg/#description
https://www.psycopg.org/psycopg3/docs/basic/install.html
`psycopg[binary,pool] ~= 3.1.9`
(Now I'm not sure if we actually use connection pools in production. The pool modifier installs the connection pool package.)

Not related to the Django upgrade, but there are other libraries that could be updated in the near future. At least,
* pymemcache 4.0.0 (no API changes)
* cachetools 5.3.1 (no API changes at a quick glance)
* celery 5.3.1 (no API changes at a quick glance)